hack/generate.sh

```diff
@@ -56,6 +54,9 @@ function generate_from_swagger() {
     lint_swagger
     generate_go_client
     generate_go_server
+    for f in models/* ; do
+        sed -i 's/Dash//g;s/Dot//g' $f
```

Comment:
Also we may want to verify that our swagger doesn't contain the words "dash" and "dot" in enums literally, because that would be seriously confusing when they suddenly disappear